pybaseball/team_batting.py

```diff
@@ -17,18 +18,15 @@ def team_batting(start_season: int, end_season: int = None, league: str = 'all',
     ind             : int : 1 if you want individual season level data
                             0 if you want a team's aggreagate data over all seasons in the query
     """
-    if start_season is None:
-        raise ValueError(
-            "You need to provide at least one season to collect data for. Try team_batting(season) or team_batting(start_season, end_season)."
-        )
-    if end_season is None:
-        end_season = start_season
 
-    fg_data = fangraphs.get_fangraphs_tabular_data_from_url(
-        _FG_TEAM_BATTING_URL.format(start_season=start_season, end_season=end_season, league=league, ind=ind)
-    )
+    warnings.warn("team_batting is deprecated in favor of FanGraphs().team_batting", DeprecationWarning)
```

Comment:
Same. I'm not sure about deprecating the API here. It's nice if it cleans up the back-end implementation, but I don't love the new API here